packages/client/mutations/CreateTaskIntegrationMutation.ts

```diff
@@ -12,8 +12,9 @@ graphql`
     task {
       ...IntegratedTaskContent_task
       integration {
-        __typename
+        # __typename
         ... on JiraIssue {
+          __typename
```

Comment:
I was playing around with removing `__typename` from the gitlab query. Will revert this change